addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -281,8 +282,11 @@ def _get_bom_data(self, bom, warehouse, product=False, line_qty=False, bom_line=
             if component['is_storable']:
                 if missing_qty := max(component['quantity'] - component['quantity_forecasted'], 0):
                     missing_qty = float_repr(missing_qty, self.env['decimal.precision'].precision_get('Product Unit'))
-                    route_name = component['route_name'] or _('Order')
-                    component['status'] = _("%(qty)s To %(route)s", qty=missing_qty, route=route_name)
+                    if not component['route_name']:
+                        component['availability_state'] = "unavailable"
```

Comment:
```suggestion
```
Looking at how `_get_availabilities()` works, seems that if no route is found the `availability_state` is always `unavailable` (unless there's enough in stock).